packaging/assets/config/default.conf.sample

```diff
@@ -21,6 +21,7 @@
 ## Stash the pid in this file when the process is running
 ## Command line default: none
 ## Init script default: /var/run/rethinkdb/<name>/pid_file (where <name> is the name of this config file without the extension)
+## Systemd default: Uses internal mechanism, do not set this paramtert.
```

Comment:
I think rather than "Systemd default:", this should say something like "Note for systemd users: Systemd uses its own internal mechanism, do not set this parameter.".
There's also a typo in "parameter".

(in general the default for systemd is the same as the command line default, but that's not specific to this option)